substra/sdk/client.py

```diff
@@ -97,24 +95,34 @@ def __init__(self, config_path=None, profile_name=None, user_path=None,
         # set current logged user if exists
         self.set_user()
 
+        if token:
+            self.set_token(token)
+
     @logit
-    def login(self):
+    def login(self, username, password):
```

Comment:
Could be renamed to authenticate? It could be worth to investigate what's the best / more common verb to describe this action.

Reply:
This would break the matching between names in the CLI and names in the SDK though.

Reply:
Yes, I was thinking to maybe rename both but I don't think it's worth it.